core/server/helm_release_test.go

```diff
@@ -0,0 +1,109 @@
+package server
+
+import (
+	"context"
+	"testing"
+
+	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
+	. "github.com/onsi/gomega"
+	pb "github.com/weaveworks/weave-gitops/pkg/api/core"
+	"github.com/weaveworks/weave-gitops/pkg/kube"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestListHelmReleases(t *testing.T) {
+	g := NewGomegaWithT(t)
+	ctx := context.Background()
+
+	c, cleanup := makeGRPCServer(k8sEnv.Rest, t)
+	defer cleanup()
+
+	_, k, err := kube.NewKubeHTTPClientWithConfig(k8sEnv.Rest, "")
+	g.Expect(err).NotTo(HaveOccurred())
+
+	appName := "myapp"
+	ns := newNamespace(ctx, k, g)
+
+	newHelmRelease(ctx, appName, ns.Name, k, g)
+
+	res, err := c.ListHelmReleases(ctx, &pb.ListHelmReleasesRequest{
+		Namespace: ns.Name,
+	})
+	g.Expect(err).NotTo(HaveOccurred())
+	g.Expect(res.HelmReleases).To(HaveLen(1))
+	g.Expect(res.HelmReleases[0].Name).To(Equal(appName))
+}
+
+func TestGetHelmRelease(t *testing.T) {
```

Comment:
are you going to test any of the `getHelmReleaseInventory` bits? it feels important to us that we are populating that bit of the returned value correct

Reply:
ah just literally saw your comment on that 👍

Reply:
My issue with testing that function... well I have to create a full secret + store + data + "i don't know what else", because unlike kustomization,  to get helm inventory, we query the api to get a helm secret, get release from the secret storage, decode + unzip (if gzipped), and finally read object from the storage. or something like that, so that would require a huge bootstrapping. I'll try it, because I would like to test it.